microreactors/s8er/tests

```diff
@@ -21,7 +21,7 @@
     type = CSVDiff
     input = 'core_2D_bison_coupled.i'
     csvdiff = 'core_2D_bison_coupled_csv.csv'
-    executable_pattern = 'blue_crab*|dire_wolf*'
+    executable_pattern = 'griffin*|blue_crab*|dire_wolf*'
```

Comment:
@GiudGiud looks like all the tests under this folder can be driven by Griffin executable. I guess this is because Griffin has all necessary moose modules available.

Reply:
It can be. 
Do you really want this though?
This means that changes in heat conduction / tensor mechanics will break the griffin recipe and require a griffin submodule update

Reply:
Not really. I have to had this to generate the new gold files. Let me revert this.